### PR TITLE
Update to 1.9.9. JB#53033

### PR DESCRIPTION
**Pull request by pvuorela**
Three commits: first removing old sources, second bringing new ones in and the last one to update packaging. New sources under a subdirectory to keep the setup cleaner. There would be an official git repository too (though main development in mercurial), but that's over 200MB of history which felt ridiculous for a command line tool.

License text included and the rpm value now matching Fedora's.

@Tomin1 @vigejolla @spiiroin

### CODE REVIEW

**Comment by vigejolla** [comment]
> There would be an official git repository too (though main development in mercurial), but that's over 200MB of history which felt ridiculous for a command line tool.

Disk space is cheap. I would just use the usual mirror + submodule approach. It would be much easier to see that the sources actually are the official ones. Especially with a security-related tool like sudo. But that's just my opinion.

**Comment by pvuorela** [comment]
Disk space wasn't the only thing but also it took quite a while to get the submodule checked out. But sure I can also do that sort of version.

**Comment by pvuorela** [comment]
Actually still even built quite fast in obs.

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "upstream"]
+	path = upstream
+	url = https://github.com/sudo-project/sudo.git
```
Comment (spiiroin) [inline .gitmodules:3]: Direct upstream without mirroring - is this intentional?

**Review by spiiroin — APPROVED** [review]
Package build and install ok. And after configuring via "visudo" using sudo worked too.

File: .gitmodules (hunk 1 above)
Comment (pvuorela) [reply .gitmodules:3]: Yea in the sense that wanted to confirm the submodule approach first before getting a mirror. Could still ask for one.

Comment (pvuorela) [reply .gitmodules:3]: Now having a mirror

**Pull request merged**